ow_plexil/src/plexil-adapter/OwInterface.cpp

```diff
@@ -336,12 +343,12 @@ void OwInterface::jointStatesCallback
       double velocity = msg->velocity[i];
       double effort = msg->effort[i];
       if (joint == Joint::antenna_pan) {
-        m_currentPan = position * R2D;
+        m_currentPan = normalize_degrees (position * R2D);
         managePanTilt (Op_PanAntenna, m_currentPan, m_goalPan, m_panStart);
         publish ("PanDegrees", m_currentPan);
      }
       else if (joint == Joint::antenna_tilt) {
-        m_currentTilt = position * R2D;
+        m_currentTilt = normalize_degrees (position * R2D);
```

Comment:
Should this be un-normalized like the tracked pan and tilt are now in the antennat pan tilt action server?

Reply:
It seems so, and it works, done.